libs/shared/src/consts/providers/credentials/provider-credentials.ts

```diff
@@ -358,6 +358,46 @@ export const expoConfig: IConfigCredentials[] = [
   ...pushConfigBase,
 ];
 
+export const pushwooshConfig: IConfigCredentials[] = [
+  {
+    key: CredentialsKeyEnum.ApplicationCode,
+    displayName: 'Application Code',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.NotificationTitle,
+    displayName: 'Notification Title',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.Subscribe,
+    displayName: 'Auto Subscribe',
+    type: 'boolean',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.UserId,
+    displayName: 'User ID',
+    type: 'string',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.Tags,
+    displayName: 'Tags',
+    type: 'object',
+    required: true,
+  },
+  {
+    key: CredentialsKeyEnum.NotificationImageURL,
+    displayName: 'Notification Image URL',
+    type: 'string',
+    required: true,
+  },
```

Comment:
This is neither required. Pushwoosh has some defaults that can be overriden.
https://github.com/Pushwoosh/web-push-notifications/blob/401489da1d845c9795383b843f8d3975eab73f13/src/constants.ts#L15